xline-client/src/lib.rs

```diff
@@ -417,9 +417,18 @@ struct AuthService<S> {
 impl<S> AuthService<S> {
     /// Create a new `AuthService`
     #[inline]
+    #[cfg(not(madsim))]
     fn new(inner: S, token: Option<Arc<HeaderValue>>) -> Self {
         Self { inner, token }
     }
+
+    /// Create a new `AuthService`
```

Comment:
/// bypassed

Reply:
I think it's better to be consistent with comment in a non-madsim environment